ref/supported-video-codecs.md

```diff
@@ -6,10 +6,8 @@ Currently, the following video codecs are supported:
 
  * H.264 - The use of H.264 requires a license from the [MPEG LA](https://www.mpegla.com/). Ensure you have the rights to stream H.264 encoded video content to your users.
  * VP8
-
-In the future we plan to add support for:
-
- * VP9
  * AV1
 
+In the future we plan to add support for VP9 as well.
```

Comment:
I am not sure if we want to explicitly call out here in the doc. But the following looks better:
```
* AV1 - Depending on the GPU model used in the deployment, AV1 hardware encoding is the default preference over H.264 on supported GPUs, such as NVIDIA Ada Lovelace Architecture-based GPUs like L4. Otherwise, it will fallback to AV1 software encoding.
```

Reply:
Good idea, I have added it along with the supported video codes list information.